nhais/main.py

```diff
@@ -0,0 +1,50 @@
+import tornado.httpclient
+import utilities.integration_adaptors_logger as log
+from requests import healthcheck_handler
+from utilities import config
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+
+def configure_http_client():
+    """
+    Configure Tornado to use the curl HTTP client.
+    """
+    tornado.httpclient.AsyncHTTPClient.configure('tornado.curl_httpclient.CurlAsyncHTTPClient')
+
+def start_tornado_server() -> None:
+
+    supplier_application = tornado.web.Application(
```

Comment:
What is `supplier` in this context?

Reply:
tornado, renamed this to tornado_application